low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -128,12 +128,12 @@ class SSPLDevDeploy:
         """Stop sspl service and remove SSPL RPMS."""
         cmd = "systemctl stop sspl-ll.service"
         _ = self._execute_cmd(cmd)
-        cmd = "yum remove -y cortx-sspl.noarch"
+        cmd = "yum --setopt=tsflags=noscripts erase -y cortx-sspl.noarch"
```

Comment:
Did you try following before above option, as it wont check dependency 
>yum clean all && rpm --rebuilddb

https://serverfault.com/questions/613256/yum-error-in-preun-scriptlet-when-removing-packages

Reply:
@shriya-deshmukh tried but above steps didnt work. So current changes are fine then